qemu/tests/nic_hotplug.py

```diff
@@ -140,7 +153,7 @@ def device_add_nic(pci_model, netdev, device_id):
                 useddevice_id = primary_nic[0].netdev_id
                 logging.info("Hot-plug NIC with the netdev already in use")
                 try:
-                    add_output = device_add_nic(nic_model, useddevice_id, nic_name)
+                    device_add_nic(nic_model, useddevice_id, nic_name)
```

Comment:
@yanan-fu Em...good point, not thought about it before, it for sure can be removed, just I would like to keep it in case it has further usage, if you are not against.

Reply:
@vivianQizhu  it is ok for me, keep them do not have side effect.